lib/puppet/type/cisco_evpn_stormcontrol.rb

```diff
@@ -0,0 +1,73 @@
+#
+# December 2017
+#
+# Copyright (c) 2017-2018 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Puppet::Type.newtype(:cisco_evpn_stormcontrol) do
+  @doc = %(Manages a Cisco Evpn stormcontrol level.
+
+  cisco_evpn_stormcontrol {'<packet_type>':
+    ..attributes..
+  }
+
+  <packet_type> is the type of packet like broadcast, unicast or multicast.
+
+  Example:
+    cisco_evpn_stormcontrol {'broadcast':
+      ensure              => present,
+      level       => '50',
+    }
+  )
+
+  ##############
```

Comment:
Is there a reason you don't use `self.title_patterns` for this type?

Reply:
This type has only one namevar. I think it's overkill to add title_patterns here.